include/mbgl/storage/default_file_source.hpp

```diff
@@ -18,8 +20,13 @@ class DefaultFileSource : public FileSource {
 
     std::unique_ptr<FileRequest> request(const Resource&, Callback) override;
 
+    // For testing purposes only.
```

Comment:
I don't think we want to add a `gtest.h` dependency to non-test headers. I tried various permutations of `friend class Storage;` and forward declarations but couldn't it to work.